src/components/Registration/RegistrationStepTwo/RegistrationStepTwo.tsx

```diff
@@ -69,7 +69,11 @@ export const RegistrationStepTwo = () => {
                       </FormGroup>
                       <p className="pt-3 mb-5">
                         {t("signUp.stepTwo.legalRep.disclaimer")}&nbsp;
-                        <a href="http://www.indicepa.gov.it">
+                        <a
```

Comment:
nitpicking here: by the point of view af accessibility, for any link that opens a new window you should provide
1. a visual hint (ie. an external link icon)
1. a written hint (ie. "open in a new window")
see:
https://www.w3.org/TR/WCAG20-TECHS/G201.html

moreover there's no way to open a link with target=blank in the same window (this does not state for the opposite: is it always possible to open a link a in a new tab).

imho the best approach is to never use target=blank, add a visual hint that the link is external and let the user chose to open it in a new tab :-)

Reply:
@alicecas